pkgs/applications/networking/browsers/firefox/default.nix

```diff
@@ -64,7 +65,6 @@ stdenv.mkDerivation rec {
       "--disable-necko-wifi" # maybe we want to enable this at some point
       "--disable-installer"
       "--disable-updater"
-      "--disable-pulseaudio"
```

Comment:
For now yes, but this can (potentially easily, I'll try this) be fixed -- we only need `pulseaudio` headers, and only at compile time. By "install" here I mean "it will be copied to /nix/store" -- nothing would be used from there is you won't tell the wrapper to do so.